thrift/src/main/thrift/atoms/sidenote.thrift

```diff
@@ -0,0 +1,58 @@
+namespace * contentatom.sidenote
+namespace java com.gu.contentatom.thrift.atom.sidenote
+#@namespace scala com.gu.contentatom.thrift.atom.sidenote
+
+include "entity.thrift"
+include "../shared.thrift"
+
+enum SideNoteType {
+  QA = 0,
+  GUIDE = 1,
+  TIMELINE = 2,
+  PROFILE = 3
+}
+
+struct SideNoteAtom {
+  1: optional string typeLabel
+  2: required string title
```

Comment:
We don't need title here as we have https://github.com/guardian/content-atom/blob/master/thrift/src/main/thrift/contentatom.thrift#L76